crates/sui-bridge/src/config.rs

```diff
@@ -226,7 +233,8 @@ pub struct BridgeClientConfig {
     pub eth_bridge_contracts: Vec<EthAddress>,
     pub sui_bridge_modules: Vec<Identifier>,
     pub eth_bridge_contracts_start_block_override: BTreeMap<EthAddress, u64>,
-    pub sui_bridge_modules_start_tx_override: BTreeMap<Identifier, TransactionDigest>,
+    /// The EventID needs to be valid, namely it exists and matches the filter. Otherwise, it will miss one event.
```

Comment:
What does `miss one event` mean?

Reply:
It's how EventID as a cursor works. Today it's implemented in a way that if the given cursor does not exist, or does not match the filter, the returned result will skip one legit event ... which I find inconvenient in many cases. But it's a separate problem to fix cc @emmazzz  @gegaowp